app/controllers/movies_controller.rb

```diff
@@ -37,10 +37,10 @@ def update
   end
 
   def show
-    @movie = Movie.friendly.find(params[:id])
-    if request.path != movie_path(@movie)
-      return redirect_to @movie, :status => :moved_permanently
-    end
+    @media = Movie.friendly.find(params[:id])
+    return redirect_to movie, status: :moved_permanently if request.path != movie_path(@media)
+
+    render 'shared/media_profile'
```

Comment:
Instead of rendering the default `show.html.erb` with a `movie_show` partial in it, I'm skipping the middle man and specifying the view I'd like to use. I do the same thing in the relevant `TmdbController` actions too. (see below)